docs/content/cookbook/1.index.md

```diff
@@ -0,0 +1,18 @@
+---
+layout: 'fullscreen'
```

Comment:
This is not visible in the sidebar. I left it as it will be useful when we transition this guide to a cookbook.